src/proxy/http/HttpConfig.cc

```diff
@@ -32,6 +32,7 @@
 #include "proxy/http/HttpConfig.h"
 #include "proxy/hdrs/HTTP.h"
 #include "iocore/eventsystem/ConfigProcessor.h"
+#include "iocore/eventsystem/UnixSocket.h"
```

Comment:
The iocore/eventsystem/SocketManager.h file was not included as it should have been; I did not forget to remove it here.